test/parallel/test-whatwg-encoding-textencoder.js

```diff
@@ -2,50 +2,93 @@
 'use strict';
 
 const common = require('../common');
-
 const assert = require('assert');
 const { TextDecoder, TextEncoder } = require('util');
 const { customInspectSymbol: inspect } = require('internal/util');
 
-const encoded = Buffer.from([0xef, 0xbb, 0xbf, 0x74, 0x65,
-                             0x73, 0x74, 0xe2, 0x82, 0xac]);
+// This test checks TextEncoder's instantiation, default encoding,
+// encoding capabilities, and conformity with util.inspect().
 
-// Make Sure TextEncoder exists
-assert(TextEncoder);
+{
+  const textEncoder = new TextEncoder();
+  assert.ok(textEncoder);
+  assert.strictEqual(textEncoder instanceof TextEncoder, true);
+}
 
-// Test TextEncoder
 {
-  const enc = new TextEncoder();
-  assert.strictEqual(enc.encoding, 'utf-8');
-  assert(enc);
-  const buf = enc.encode('\ufefftest€');
-  assert.strictEqual(Buffer.compare(buf, encoded), 0);
+  const textEncoder = new TextEncoder();
+  assert.strictEqual(textEncoder.encoding, 'utf-8');
+}
+
+function checkIfBuffersAreEqual(buffer1, buffer2) {
+  const comparisonResult = Buffer.compare(buffer1, buffer2);
+  return comparisonResult === 0;
```

Comment:
This is strictly less legible than using `Buffer.compare` where appropriate.

Reply:
How?